src/Core/Components/Image/FluentImage.razor.cs

```diff
@@ -0,0 +1,93 @@
+﻿// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The Fluent Image component is used to display an image, providing various options for styling and layout.
+/// </summary>
+public partial class FluentImage : FluentComponentBase
+{
+    /// <summary/>
+    protected string? ClassValue => DefaultClassBuilder
+        .Build();
+
+    /// <summary/>
+    protected string? StyleValue => DefaultStyleBuilder
+        .Build();
+
+    /// <summary/>
+    private static string? ImageClassValue => new CssBuilder("fluent-image-item")
+        .Build();
+
+    /// <summary/>
+    private string? ImageStyleValue => new StyleBuilder()
+        .AddStyle("width", Width, () => !string.IsNullOrEmpty(Width))
+        .AddStyle("height", Height, () => !string.IsNullOrEmpty(Height))
+        .Build();
+
+    /// <summary>
+    /// The height of the image.
```

Comment:
The height of the image, **applies only to the <see cref="Source"/> parameter.**